### PR TITLE
Guide V1 Updates

### PR DESCRIPTION
**Pull request by AnujanM**

### CODE REVIEW

File: firstyearguide/index.md [hunk 1]
```diff
@@ -140,23 +150,37 @@ So you’ve gotten into the courses you like, and you’ve planned out your time
...
+   Also, make sure that your code runs on the lab computers (this may sound crazy, but people have submitted assignments that error out because of the following two things):
+   1. They didn’t submit the right file or it had an error, or
+   2. Their computer inputted a random non ASCII character and the lab couldn't parse that file, resulting in a 0 because it didn’t run. 
+
```
Comment (ansermino) [inline firstyearguide/index.md]: Something such as `3. The teach cs machines are Linux. If you write your code on a Mac or Windows machine, or even a Linux machine with a different configuration your code may depend on libraries or encoding that do not match the teach.cs machines`. Perhaps this is what you intended for 2, specifically line break characters

File: firstyearguide/index.md [hunk 2]
```diff
@@ -140,23 +150,37 @@ So you’ve gotten into the courses you like, and you’ve planned out your time
...
+So after reading this, most students panic and ask: “What's the cutoff?” Well, we wish we could give you an answer, and we wish that the department could give an answer, but it’s not that easy. 
+
+- The cutoff is not an arbitrary number, say 60 or 75, like some other programs. In reality, the cutoff for CS is the top x of student averages from 148 and 165 that have applied to the CS program, where x is the number of spots in the program for that year.
```
Comment (ansermino) [inline firstyearguide/index.md]: Top x number or #

File: firstyearguide/index.md [hunk 3]
```diff
@@ -188,26 +214,35 @@ The Professional Experience Year Co-op Program, administered by the Engineering
 
 Click on any of the links to go there:
 - [CS Community Slack](https://uoftcscommunity.slack.com)
```
Comment (ansermino) [inline firstyearguide/index.md]: Could we add a note for this one that they need to click create account at the bottom and use a uoft email?

**Review by ansermino — CHANGES_REQUESTED** [review]
Looks great! Minor edits

File: firstyearguide/index.md (hunk 1 above)
Comment (AnujanM) [reply firstyearguide/index.md]: OK i've added yours as a seperate line but 2's intention was for random non-english characters and non-visible 2 byte charaters that came due to some weird copy paste.

File: firstyearguide/index.md (hunk 2 above)
Comment (AnujanM) [reply firstyearguide/index.md]: Fixed

File: firstyearguide/index.md (hunk 3 above)
Comment (AnujanM) [reply firstyearguide/index.md]: Sure Added

**Review by ansermino — APPROVED** [review]

**Pull request merged**